usr/bin/whonix-gateway-firewall

```diff
@@ -411,17 +414,24 @@ ipv4_input_rules() {
       fi
 
       local local_port_to_open
+      if test -n "${EXTERNAL_OPEN_PORTS}" && [ "${info_enabled}" = "1" ]; then
```

Comment:
Here `"${info_enabled}" = "1"` makes sense. Good as is. Meaning here it's good to not flood users with too much information.

Reply:
But `test -n "${EXTERNAL_OPEN_PORTS}"` could be dropped? When using `info_enabled=1` anyhow, I guess it's ok to be more verbose and point out that there are no `EXTERNAL_OPEN_PORTS`?

Reply:
https://github.com/Whonix/whonix-firewall/pull/1/commits/3c5fd6b4eb446cd997384818bee35d2baf0629bc